roles/openshift_aws/tasks/iam_role.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create an iam role
+  iam_role:
+    name: "{{ item.value.iam_role }}"
```

Comment:
This looks like it will fail against node group types other than master, but appears it will run for all node group types.

Reply:
I placed an `iam_role` section for each node group type.  We want all node group types to have instance profiles so that AMIs do not require baking credentials into the openshift-node.